server/main.tf

```diff
@@ -47,7 +47,7 @@ provider "aws" {
 
 # data "archive_file" "lambda_greetings_server" {
 #   type        = "zip"
-#   source_dir  = "${path.module}/lambda"
+#   source_dir  = "${path.modul}/lambda"
```

Comment:
I know this is a comment out line, but maybe 'modul' was changed by accident too, maybe '**module**' is the correct one?

Reply:
@ianliuwk1019 It has been fixed.

Thanks.